lib/recommendation-server.js

```diff
@@ -48,6 +49,21 @@ RecommendationServer.prototype = {
   _url: function(query) {
     return `${this.server}/?q=${encodeURIComponent(query)}`;
   },
+  warm: function() {
+    /*
+    Fire off a dummy request to the recommendation server to warm an HTTP/2
+    connection to take advantage of multiplexing.
+    */
+    const req = this.request;
+    if (!req.isInFlight) {
+      req.addEventListener('loadend', evt => {
+        req.isInFlight = false;
+      });
+      req.open('GET', `${this.server}/__lbheartbeat__`);
```

Comment:
The dockerflow docs don't say whether the lbheartbeat endpoint is public or just intended to be exposed to the ELBs. Assuming it'll be public for now, but wanted to call this out.